server/src/api/post.rs

```diff
@@ -164,12 +176,34 @@ impl Perform<PostResponse> for Oper<CreatePost> {
       Err(_e) => return Err(APIError::err("couldnt_find_post").into()),
     };
 
-    Ok(PostResponse { post: post_view })
+    if let Some(rl) = &rate_limit_info {
+      rl.rate_limiter
+        .lock()
+        .unwrap()
+        .check_rate_limit_post(&rl.ip, false)?;
```

Comment:
and why is the rate limit checked a second time here?

Reply:
Comment above.